MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
Interestingly the MS C# style guide used for the new .NET core project doesn't mention internals:

https://github.com/dotnet/corefx/wiki/Coding-style

So looking thru their code base I was able to find one example case:

https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/src/System/Diagnostics/ThreadInfo.cs#L15

So I would say yeah... rename it.